### PR TITLE
Frontpage in Norwegian

### PR DESCRIPTION
**Pull request by Devalo**
Added the necessary config setups and translated the frontpage index.md to norwegian.

### CODE REVIEW

File: no/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+[![License](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
+
+Leksjoner i programmeringsspråket Elixir, inspirert av Twitters [Scala Skole](http://twitter.github.io/scala_school/).
+
+
+_Du oppfordres til å delta, samt gi tilbakemeldinger!_
+
+## Om Elixir
+
+"Elixir er et dynamisk, funksjonelt språk utviklet for å bygge skalerbar og vedlikeholdsvennlige applikasjoner." [elixir-lang.org](http://elixir-lang.org/)
```
Comment (SirSJR) [inline no/index.md]: "for å bygge skalerbare og vedlikeholdsvennlige applikasjoner"

Du glemte 'e' i slutten av skalerbar.

Comment (Devalo) [inline no/index.md]: Thank you for this. 
It will be fixed in my next pull.

Comment (doomspork) [inline no/index.md]: @Devalo do you want to make this change before I merge?

**Comment by doomspork** [comment]
Thank you @Devalo & @SirSJR! 

What are your twitter handles?

**Comment by SirSJR** [comment]
Hey @doomspork my twitter handle is: therobinsj

**Comment by Devalo** [comment]
@doomspork mine is @ScopeSV

**Pull request merged**